install_config/topics/sssd_for_ldap_failover_configure_sssd.adoc

```diff
@@ -51,8 +51,40 @@ and add this attribute:
 ----
 [domain/example.com]
 ...
-ldap_user_extra_attrs = mail
+ldap_user_extra_attrs = mail <1>
 ----
+<1> Specify the correct attribute to retrieve email addresses for your LDAP
+solution. For IPA, specify `mail`. Other LDAP solutions might use another
+attribute, such as `email`.
+
+. Modify the *_domain_* parameter in the *_/etc/sssd/sssd.conf_* file to contain
+only one domain name.
++
+----
+domains = your.domain.name
+----
+
+. Confirm that the mail attribute you specified returns an email address from
+your domain:
++
+----
+# dbus-send --print-reply --system --dest=org.freedesktop.sssd.infopipe \
```

Comment:
This probably will not work until after you restart SSSD, which is a later step.